### ISSUE TITLE
pyright reports errors in non-english language

### ISSUE THREAD

**Issue by denballakh** [microsoft/pyright#7438]
**Describe the bug**
```py
# _.py
x: int = ''
```
```bat
D:\>pyright --version
pyright 1.1.353

D:\>pyright _.py
d:\_.py
  d:\_.py:1:10 - error: Выражение типа "Literal['']" нельзя присвоить объявленному типу "int"
    "Literal['']" несовместим с "int" (reportAssignmentType)
1 error, 0 warnings, 0 informations
```
Pyright provides translated error messages instead of regular ones.

**Expected behaviour**
Error messages are in english by default.

**Some info**
- Windows 10
- CPython3.12
- pyright `1.1.353` (i ran `pip install pyright -U`, so it is the latest version at the moment)
- also can reproduce in `LSP-pyright` plugin of Sublime Text editor, it uses the same executable under the hood

**Comment by erictraut** [comment]
This is by design. Refer to [this documentation](https://microsoft.github.io/pyright/#/configuration?id=locale-configuration) for details.
